install/Makefile

```diff
@@ -6,34 +6,46 @@ RELEASE_VERSION ?= latest
-	$(CP) ./cluster-operator/048-Crd-kafkamirrormaker2.yaml ../helm-charts/strimzi-kafka-operator/templates/048-Crd-kafkamirrormaker2.yaml
-	$(CP) ./cluster-operator/049-Crd-kafkarebalance.yaml ../helm-charts/strimzi-kafka-operator/templates/049-Crd-kafkarebalance.yaml
-	oldpath=`pwd` && cd ../helm-charts/strimzi-kafka-operator/templates/ &&  $(FIND) . -type f -name '04*-Crd-*.yaml' -exec yq write -i {} metadata.labels.app "{{ template \"strimzi.name\" . }}" \; && cd $$oldpath
-	oldpath=`pwd` && cd ../helm-charts/strimzi-kafka-operator/templates/ &&  $(FIND) . -type f -name '04*-Crd-*.yaml' -exec yq write -i {} metadata.labels.chart "{{ template \"strimzi.chart\" . }}" \; && cd $$oldpath
-	yq write -i ../helm-charts/strimzi-kafka-operator/templates/040-Crd-kafka.yaml metadata.labels.component "kafkas.kafka.strimzi.io-crd"
-	yq write -i ../helm-charts/strimzi-kafka-operator/templates/041-Crd-kafkaconnect.yaml metadata.labels.component "kafkaconnects.kafka.strimzi.io-crd"
-	yq write -i ../helm-charts/strimzi-kafka-operator/templates/042-Crd-kafkaconnects2i.yaml metadata.labels.component "kafkaconnects2is.kafka.strimzi.io-crd"
-	yq write -i ../helm-charts/strimzi-kafka-operator/templates/043-Crd-kafkatopic.yaml metadata.labels.component "kafkatopics.kafka.strimzi.io-crd"
-	yq write -i ../helm-charts/strimzi-kafka-operator/templates/044-Crd-kafkauser.yaml metadata.labels.component "kafkausers.kafka.strimzi.io-crd"
-	yq write -i ../helm-charts/strimzi-kafka-operator/templates/045-Crd-kafkamirrormaker.yaml metadata.labels.component "kafkamirrormakers.kafka.strimzi.io-crd"
-	yq write -i ../helm-charts/strimzi-kafka-operator/templates/046-Crd-kafkabridge.yaml metadata.labels.component "kafkabridges.kafka.strimzi.io-crd"
-	yq write -i ../helm-charts/strimzi-kafka-operator/templates/047-Crd-kafkaconnector.yaml metadata.labels.component "kafkaconnectors.kafka.strimzi.io-crd"
-	yq write -i ../helm-charts/strimzi-kafka-operator/templates/048-Crd-kafkamirrormaker2.yaml metadata.labels.component "kafkamirrormaker2.kafka.strimzi.io-crd"
-	yq write -i ../helm-charts/strimzi-kafka-operator/templates/049-Crd-kafkarebalance.yaml metadata.labels.component "kafkarebalances.kafka.strimzi.io-crd"
-
-	oldpath=`pwd` && cd ../helm-charts/strimzi-kafka-operator/templates/ &&  $(FIND) . -type f -name '04*-Crd-*.yaml' -exec yq write -i {} metadata.labels.release "{{ .Release.Name }}" \; && cd $$oldpath
-	oldpath=`pwd` && cd ../helm-charts/strimzi-kafka-operator/templates/ &&  $(FIND) . -type f -name '04*-Crd-*.yaml' -exec yq write -i {} metadata.labels.heritage "{{ .Release.Service }}" \; && cd $$oldpath
-	oldpath=`pwd` && cd ../helm-charts/strimzi-kafka-operator/templates/ &&  $(FIND) . -type f -name '04*-Crd-*.yaml' -exec $(SED) -i -e '1s/^/{{- if .Values.createGlobalResources -}}\n/' -e '$$s/$$/\n{{- end -}}/' {} \; && cd $$oldpath
+	for HELM_VERSION in 2 3	; do	\
```

Comment:
Something in this block does not work - it throws error in Travis which is not handled properly somewhere and the build continues. But the `yq` is not used properly here somewhere- I wonder if this can be done somehow more elegantly. Right now it is one huge command. So from the error you don't even know which exact lines is causeng the issue. 

If the files the same for Helm 2 and Helm 3, would it be easier to render it once e.g. for Helm3 and than just copy it to Helm2?

Reply:
I had massive problems with loops in a recipe. I thought that it would be more consistent to keep everything in a loop, but I will follow your suggestion in order to finally get Travis to pass.